CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ This file is influenced by http://keepachangelog.com/.
 
 ### Fixed
 
+- Fixed visual defect with homepage category list on IE10/11
```

Comment:
If we change this to say "Improved IE support" then we don't have to keep updating the changelog all week for minor things.